src/translation_table.py

```diff
@@ -0,0 +1,267 @@
+from bs4 import BeautifulSoup
+
+
+EN = (
+    {
+        'variable': 'congressperson_name',
+        'name': 'Congressperson Name',
+        'desc': """Name used by the congress person during his term in
+            office. Usually it is composed by two elements: a given name and a
+            family name; two given names; or two forename, except if the head
+            of the Lower House explicitly alter this rule in order to avoid
```

Comment:
> From what I understand: Câmara dos Deputados/Chamber of Deputies is the lower house, while Senado/Senate is the upper.

Lower and Upper Houses are the general names. Each country have their own names and reason for it (House of Representatives / Senate in the US if I'm not wrong; House of Commons and House of Lords in UK etc.).

I tend to prefer using the general term because it is clearer for those who don't know the specificities of Brazilian political structure. But saying they refer themselves as _Chamber of Deputies_ is indeed a strong argument — let's embrace it.